sdks/java/io/hadoop-input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/HIFIOWithCassandraTest.java

```diff
@@ -0,0 +1,207 @@
+import org.apache.beam.sdk.transforms.Count;
+import org.apache.beam.sdk.transforms.SimpleFunction;
+import org.apache.beam.sdk.values.KV;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.mapreduce.InputFormat;
+import org.cassandraunit.utils.EmbeddedCassandraServerHelper;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.junit.runners.MethodSorters;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.mapping.annotations.Column;
+import com.datastax.driver.mapping.annotations.Table;
+
+/**
+ *
+ * Tests to validate HadoopInputFormatIO for embedded Cassandra instance.
+ *
+ */
+@RunWith(JUnit4.class)
+@FixMethodOrder(MethodSorters.JVM)
+public class HIFIOWithCassandraTest implements Serializable {
+	private static final long serialVersionUID = 1L;
+	private static final String CASSANDRA_KEYSPACE = "hif_keyspace";
+	private static final String CASSANDRA_HOST = "127.0.0.1";
+	private static final String CASSANDRA_TABLE = "person";
+	private static transient Cluster cluster;
+	private static transient Session session;
+	@Rule
+	public final transient TestPipeline p = TestPipeline.create();
+
+	@BeforeClass
+	public static void startEmbeddedCassandra() throws Exception {
+		EmbeddedCassandraServerHelper.startEmbeddedCassandra("/cassandra.yaml",
+				"target/cassandra", Long.MAX_VALUE);
+		cluster = Cluster.builder().addContactPoint(CASSANDRA_HOST)
+				.withClusterName("beam").build();
+		session = cluster.connect();
+	}
+
+	@Before
+	public void createTable() throws Exception {
+		session.execute("CREATE KEYSPACE "
+				+ CASSANDRA_KEYSPACE
+				+ " WITH REPLICATION = {'class':'SimpleStrategy', 'replication_factor':1};");
+		session.execute("USE " + CASSANDRA_KEYSPACE);
+		session.execute("CREATE TABLE person(person_id int, person_name text, PRIMARY KEY(person_id));");
+		session.execute("INSERT INTO person(person_id, person_name) values(0, 'John Foo');");
+		session.execute("INSERT INTO person(person_id, person_name) values(1, 'David Bar');");
+	}
+
+	/**
+	 * Test to read data from embedded Cassandra instance and verify whether
+	 * data is read successfully.
+	 *
+	 * @throws Exception
+	 */
+	@Test
+	public void testHIFReadForCassandra() throws Exception {
+		Pipeline p = TestPipeline.create();
+		Configuration conf = getConfiguration();
+		SimpleFunction<Row, String> myValueTranslate = new SimpleFunction<Row, String>() {
+			@Override
+			public String apply(Row input) {
+				return input.getString("person_name");
+			}
+		};
+		PCollection<KV<Long, String>> cassandraData = p
+				.apply(HadoopInputFormatIO.<Long, String> read()
```

Comment:
can we apply this formatting to the same line in HIFIOElasticIT & HIFIOCassandraIT? we do this exact same line in those tests, but this one is so much more readable.